src/porepy/numerics/ad/equation_manager.py

```diff
@@ -121,16 +120,20 @@ def __init__(
         self._identify_discretizations()
 
     # TODO why is this called local? and not global?
-    def local_dofs(self, true_ad_variables: Optional[list] = None) -> np.ndarray:
-        if true_ad_variables is None:
+    def local_dofs(self, active_variables: Optional[list] = None) -> np.ndarray:
```

Comment:
1) Please use List, not list (the latter may not be properly recognized, from what I know, it was not properly introduced before Py 3.9).
2) Specify type of list, I suppose `List[str]`